bundles/io/org.eclipse.smarthome.io.console.karaf/META-INF/MANIFEST.MF

```diff
@@ -7,9 +7,9 @@ Bundle-SymbolicName: org.eclipse.smarthome.io.console.karaf
 Bundle-Vendor: Eclipse.org/SmartHome
 Bundle-Version: 0.10.0.qualifier
 Import-Package: 
- org.apache.karaf.shell.api.action.lifecycle;version="4.0.0",
- org.apache.karaf.shell.api.action;version="4.0.0",
- org.apache.karaf.shell.api.console;version="4.0.0",
+ org.apache.karaf.shell.api.action.lifecycle,

```

Comment:
IIRC I added this sometime because it has been tested with >= 4.0.0 only. Do you test that is works with 3.x, too?
So, to be clear, it has been added by intention and not because the Eclipse IDE added it...